superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -274,19 +287,10 @@ class SaveModal extends React.Component<SaveModalProps, SaveModalState> {
         {(this.state.alert || this.props.alert) && (
           <Alert
```

Comment:
Not sure if this was like this before as well, but this alert looks weird in your video. The text is not horizontally aligned, looks like it is missing an icon on the left

Reply:
Sorry, that was because I made the error show up by disabling my network connection and it made the icon not load.  Here's what it looks like when I cause an error to happen via other methods:
![localhost_9000_explore__form_data_key=lCqno0lpPD4fU4rtWGdzKzKtvJ4w-qyCaNbpdTrF3PsptagPSiXi1LadPHLg_asV slice_id=210](https://user-images.githubusercontent.com/13007381/191022114-915c9509-9ddb-40e0-9773-a8307a455950.png)